### ISSUE TITLE
maxFontSize trips up the calculation after 100

### ISSUE THREAD

**Issue by Willadaygo** [trys/postcss-utopia#5]
So I've been trying to use the epic plugin you've made, I add this in my CSS:

```postcss
.type-giga {
  font-size: utopia.clamp(30, 99, 320, 1440); // spits out clamp(1.875rem, 0.6429rem + 6.1607vi, 6.1875rem);
}
```

but then if I add any number from 100 to 299, it doesn't seem to work. Looks like it flips the font-size number in the clamp that gets generated 🤪

```postcss
.type-giga {
  font-size: utopia.clamp(30, 120, 320, 1440); // spits out clamp(7.5rem, 0.2679rem + 8.0357vi, 1.875rem);
}
```

Any idea what I'm doing wrong?

**Comment by trys** [comment]
Ooh, interesting! I'll take a look today and get back to you

**Comment by trys** [comment]
@Willadaygo Thanks for the bug report - the fix has been released to `v1.1.0`

**Comment by Willadaygo** [comment]
legend nice one, big love!
